routes/company-routes.js

```diff
@@ -1,105 +0,0 @@
-const express = require("express");
-const router = express.Router();
-const Review = require("../models/review");
-const User = require("../models/user");
```

Comment:
You have a few unused imports in various files, like this `User` here. It's always good practice to remove unused imports